lib/helpers/contacts/Recipients.php

```diff
@@ -0,0 +1,102 @@
+<?php
+/**
+ * This helper builds a html form and provides a submission endpoint for the form that makes a /contactdb/recipients API call.
+ *
+ * PHP version 5.6, 7
+ *
+ * @author    Kraig Hufstedler <kraigory@gmail.com>
+ * @copyright 2017 SendGrid
+ * @license   https://opensource.org/licenses/MIT The MIT License
+ * @version   GIT: <git_id>
+ * @link      http://packagist.org/packages/sendgrid/sendgrid
+ */
+namespace SendGrid;
+
+class RecipientForm
+{
+    public $html;
+
+    public function __construct($url)
+    {
+        $html = '<form action="' . $url . '" method="post">
+        First Name: <input type="text" name="first-name"><br>
+        Last Name: <input type="text" name="last-name"><br>
+        E-mail: <input type="text" name="email"><br>
+        <input type="submit">
+        </form>';
+        $this->setHtml($html);
+    }
+
+    public function setHtml($html)
+    {
+        $this->html = $html;
+    }
+
+    public function getHtml()
+    {
+        return $this->html;
+    }
+
+}
+
+/**
+ * Class for creating a recipient
+ */
+class Recipient implements \JsonSerializable
+{
+    public $firstName;
```

Comment:
Same question as above for the public properties, doing this breaks OOP ecapsulation.

Reply:
Will fix.